erpnext/public/js/controllers/buying.js

```diff
@@ -347,6 +347,12 @@ erpnext.buying.get_items_from_product_bundle = function(frm) {
 			freeze: true,
 			callback: function(r) {
 				if(!r.exc && r.message) {
+					if($.isArray(frm.doc.items) && frm.doc.items.length > 0) {
+						// remove first item row if empty
+						if(!frm.doc.items[0].item_code) {
```

Comment:
This is actually a material request. However, at second glance, I'm not too comfortable with the fix. I'm thinking of another fix. I'll reopen once I cook up something nicer.